<?xml version="1.0" encoding="UTF-8"?>
<launch>
    <arg name="image_src" default="/sensing/camera/traffic_light/image_raw" />
    <arg name="camera_info_src" default="/sensing/camera/traffic_light/camera_info" />
    <arg name="camera_id" default="/" />
    <arg name="intrinsics_file" default="/" />
    <arg name="target_frame" default="velodyne"/>
    <arg name="camera_frame" default="camera"/>

    <node pkg="image_view2" type="image_view2" name="image_view2" ns="$(arg camera_id)" output="screen">
        <remap from="image" to="$(arg image_src)" />
    </node>

    <node pkg="autoware_camera_lidar_calibrator" type="autoware_camera_lidar_calibration_node" name="autoware_camera_lidar_calibration_node" ns="$(arg camera_id)" output="screen">
        <param name="image_src" value="$(arg image_src)" />
        <param name="camera_info_src" value="$(arg camera_info_src)" />
    </node>

    <!--pubtf-->		
    <include file="$(find autoware_camera_lidar_calibrator)/launch/pubtf.launch">
    </include>
</launch>
